main.go

```diff
@@ -15,8 +15,8 @@ var (
 
 type videoBridgeStats struct {
 	Threads                               int     `json:"threads"`
-	BitRateDownload                       int     `json:"bit_rate_download"`
-	BitRateUpload                         int     `json:"bit_rate_upload"`
+	BitRateDownload                       float64     `json:"bit_rate_download"`
```

Comment:
can you `go fmt` to adjust the code style?

Reply:
Done, sorry was in a hurry :)